### PR TITLE
Add `customEngineAgents` to devPreview manifest reference

### PR DESCRIPTION
**Pull request by erikadoyle**
Reference doc update for https://github.com/microsoft/json-schemas/pull/272/files

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e1edec3fcc9650c12b6419983ad8c659be76d8f9/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 137 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1cd176b2-5d8d-4a51-82bc-3dd2756b380d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0bcff3c48643531af4bcb602adb702fd6b156ed3/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 136 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35f02018-728b-4860-955b-953e1787acb6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bbd1c2c9c63df970e1ef6c1e607c404932b03317/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 136 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7e165220-0084-4bbf-8d6b-db6121184569) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -531,11 +531,12 @@ Describes a set of mutual dependencies between two or more app capabilities. A M
 
 **Optional** &ndash; Object
 
-Defines one or more agents to Microsoft 365 Copilot (formerly known as `copilotExtensions`). [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models (formerly known as `declarativeCopilots`).
+Defines one or more agents to Microsoft 365 Copilot (formerly known as `copilotExtensions`). [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models (formerly known as `declarativeCopilots`). [Custom engine agents](/microsoft-365-copilot/extensibility/overview-custom-engine-agent) are conversational Teams bots that use custom AI language models and orchestration, yet are selectable (along with installed declarative agents) as **Agents** from the Microsoft 365 Copilot side panel.
```
Comment (v-preethah) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:534]: 1. ...Microsoft 365 Copilot that run on its same orchestrator.... - Must be fixed to ....Microsoft 365 Copilot that run on the same orchestrator...

2.  ...as **Agents** from the Microsoft 365 Copilot side panel. - Is **Agents** mentioned here is an UI term to have it in bold?

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 2]
```diff
@@ -546,6 +547,19 @@ Represents a customization of Microsoft 365 Copilot, as defined by its manifest
 |`id`|String| |✔️| Unique identifier for the agent. When using Microsoft Copilot Studio to build agents, this is auto-generated. Otherwise, manually assign the value according to your own conventions or preference. |
 |`file`|String| |✔️| Relative path within the app package to the [declarative agent manifest](/microsoft-365-copilot/extensibility/declarative-agent-manifest) file. |
 
+### customEngineAgents
+
+> [!NOTE]
+> Custom engine agents support in Microsoft 365 Copilot is currently in limited private preview; not all developers will have access during the staged rollout.
```
Comment (v-preethah) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:553]:
```suggestion
> Custom engine agents support in Microsoft 365 Copilot is currently in limited private preview and not all developers have access during the staged rollout.
```

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 3]
```diff
@@ -546,6 +547,19 @@ Represents a customization of Microsoft 365 Copilot, as defined by its manifest
...
+
+Represents a conversational Teams bot that uses custom AI language models and orchestration, surfaced as an agent in the Microsoft 365 Copilot UI.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`|String| |✔️| Unique (bot) identifier for the custom engine agent.  Must match the `id` specified in the `bots` section of the manifest, and be of *personal* scope. |
+|`type`|String| |✔️| Type of the custom engine agent. Supported values: `bot`. |
```
Comment (v-preethah) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:560]:
```suggestion
|`type`|String| |✔️| Type of the custom engine agent. Supported value: `bot` |
```

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 4]
```diff
@@ -546,6 +547,19 @@ Represents a customization of Microsoft 365 Copilot, as defined by its manifest
...
+### customEngineAgents
+
+> [!NOTE]
+> Custom engine agents support in Microsoft 365 Copilot is currently in limited private preview; not all developers will have access during the staged rollout.
+
+Represents a conversational Teams bot that uses custom AI language models and orchestration, surfaced as an agent in the Microsoft 365 Copilot UI.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`|String| |✔️| Unique (bot) identifier for the custom engine agent.  Must match the `id` specified in the `bots` section of the manifest, and be of *personal* scope. |
```
Comment (v-preethah) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:559]:
```suggestion
|`id`|String| |✔️| Unique (bot) identifier for the custom engine agent. Must match the `id` specified in the `bots` section of the manifest and be of `personal` scope. |
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/73bfb2a91f9834660f9802ec3f15ef80cfc9498b/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 136 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f1e97e2-09e7-4a3c-87ec-255a80fdc22f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fd85297881326aef2d1a8a6ded232852dab8607f/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a5d7bde-d985-44b9-9468-b8241a97a14b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 1 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:534]: Fixed (1), good catch!
For (2), yes **Agents** is the UI label in the M365 Copilot side panel where installed agents are selectable.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e522e4f677abfc96119af998aaab7c387c34e615/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 135 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/14130943-c14f-418b-b31b-ae08ca8700b2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39f9e4c6d0ed824ec0d50ff144d6ef75ffad2b3e/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 86 | 129 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/542e19f2-a22c-48fb-84fa-21686553ed71) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**